<launch>

	<node pkg="trilateration" type="trilateration_node" name="trilateration_node" output="screen">
		<param name="global_frame_name" value="world"/>
		<param name="range_topic_name" value="/gtec/toa/ranging"/>
        <param name="anchors_topic_name" value="/gtec/toa/anchors"/>
		<param name="position_topic_name" value="/gtec/trilateration"/>
		<param name="loop_rate" value="10.0"/>
		<param name="dim" value="2"/>
	</node>

</launch>
